xarray/core/arithmetic.py

```diff
@@ -3,9 +3,18 @@
 
 import numpy as np
 
+# _typed_ops.py is a generated file
+from ._typed_ops import (
+    DataArrayGroupByOpsMixin,
+    DataArrayOpsMixin,
+    DatasetGroupByOpsMixin,
+    DatasetOpsMixin,
+    VariableOpsMixin,
+)
+from .common import ImplementsArrayReduce, ImplementsDatasetReduce
+from .ops import IncludeCumMethods, IncludeNumpySameMethods, IncludeReduceMethods
 from .options import OPTIONS, _get_keep_attrs
 from .pycompat import dask_array_type
-from .utils import not_implemented
```

Comment:
That was the only use of `not_implemented` - could you delete that from `utils`.

Reply:
:heavy_check_mark: